docs/bare-metal/README.md

```diff
@@ -0,0 +1,298 @@
+
+
+edit `/etc/sysconfig/network-scripts/ifcfg-eno1`
+
+You can leave everything that is in there but you need to to change  or add
+the following. BOOTPROTO and ONBOOT are probably already there.
+
+    BOOTPROTO="static"
+    IPADDR="172.16.222.6"
+    GATEWAY="172.16.222.1"
+    NETMASK="255.255.255.0"
+    DNS1="8.8.8.8"
+    DNS2="208.67.222.222"
+    NM_CONTROLLED=no
+    ONBOOT="yes"
+
+the dns lines are going to have to change once consul is up.
+
+NOTE: in centos 7 /etc/resolv.conf is a generated file.
+
+You could also put the dns lines in /etc/sysconfig/network.
+
+permanently change your host name with
+
+    hostnamectl set-hostname edge22
+
+After saving then finally:
+
+    systemctl restart network
+
+
+### Create Partion For Docker LVM
+
+ * su
+ * parted /dev/sda print
+ * fdisk /dev/sda
+ * Command: n
+ * partion : default
+   * please note which partition it is in.  So if its partition 5, eventually you will need to tell mantl /dev/sda5 for the LVM
+   * you kinda want all your machines to use the same partition because this partition is entered as a ssytem wide variable.
+ * first sector: default
+ * last sector: +50G
+ * Command: w
+
+ * reboot
+
+
+don't put a file system on the partion.
+
+Note that I am creating a partion size 50 Gigs, this is for docker.  Just make it consistent across your cluster.
+
+if you have MS-dos partions make it a primary partion, not an extended partion or it won't work.
+
+Additionally.  If you hang on TASK "lvm | create volume group" then you will have to apply the patch
+https://github.com/ansible/ansible-modules-extras/issues/1504 the file you need to do this to is in
+mantl base directory /library/lvg.py
+
+
+Do all the computers to this point.
+
+## Creating Your Inventory
+
+Here is an example inventory file. It should be place in the root of the mantl directory.
+
+    [role=control]
+    control01 private_ipv4=172.16.222.6 ansible_ssh_host=172.16.222.6
+    control02 private_ipv4=172.16.222.7 ansible_ssh_host=172.16.222.7
+    control03 private_ipv4=172.16.222.8 ansible_ssh_host=172.16.222.8
+
+    [role=control:vars]
+    consul_is_server=true
+    lvm_physical_device=/dev/sda3
+
+    [role=worker]
+    resource01 private_ipv4=172.16.222.11 ansible_ssh_host=172.16.222.11
+    resource02 private_ipv4=172.16.222.12 ansible_ssh_host=172.16.222.12
+    resource03 private_ipv4=172.16.222.13 ansible_ssh_host=172.16.222.13
+
+    [role=worker:vars]
+    consul_is_server=false
+    lvm_physical_device=/dev/sda3
+
+    [role=edge]
+    edge01 private_ipv4=172.16.222.16 ansible_ssh_host=172.16.222.16
+    edge02 private_ipv4=172.16.222.17 ansible_ssh_host=172.16.222.17
+
+    [role=edge:vars]
+    consul_is_server=false
+    lvm_physical_device=/dev/sda3
+
+    [dc=dc1]
+    control01
+    control02
+    control03
+    resource01
+    resource02
+    resource03
+    edge01
+    edge02
+
+I had to add the ansible_ssh_host line to run `playbooks/reboot-hosts.yml`
```

Comment:
Should we make this part of the PR? If it is not suitable for non-bare-metal use cases, we could just add a `when: if provider is defined and provider == "bare-metal"`.